### PR TITLE
homework/1

### PR DESCRIPTION
**Pull request by ghost**
Please let me know if anything need to be changed

### CODE REVIEW

File: BMI Calculator/BMI Calculator/Program.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            UserInfo();
+            UserInfo();
+        }
+
+        static void CalBMI(double weight, double height, string fName, string lName)
```
Comment (AlmightyLks) [inline BMI Calculator/BMI Calculator/Program.cs:14]: Unnecessary shortage of naming. Calling it `CalculateBMI()` is fine 😜 
Also. When I'd call a method called `CalculateBMI()`, I'd expect it to only calculate it for me and return the result to me.
Separate those processes of "CalculateX()" and printing something into different methods / places 😄

File: BMI Calculator/BMI Calculator/Program.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+        static void PrintUserInfo(string fName, string lName, double age, double weight, double height)
+        {
+            string userInfo = $"{fName} {lName} is {age} years old, his weight is {weight} kg and his height is {height} cm";
+            Console.WriteLine(userInfo);
+
+            CalBMI(weight, height, fName, lName);
+        }
+
+        static void UserInfo()
```
Comment (AlmightyLks) [inline BMI Calculator/BMI Calculator/Program.cs:28]: Method names should at the very least contain a verb. 
What is the method going to do? Perhaps `QueryUserInfo()` or `PromptUserInfo()`?

**Review by AlmightyLks — CHANGES_REQUESTED** [review]

File: BMI Calculator/BMI Calculator/Program.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Dynamic;
+
+namespace BMI_Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            UserInfo();
```
Comment (Almantask) [inline BMI Calculator/BMI Calculator/Program.cs:10]: UserInfo is a noun, method names should be verbs (actions, not things)

**Review by Almantask — CHANGES_REQUESTED** [review]
Try to change a target branch to master.

**Comment by ghost** [comment]
> 
> 
> Try to change a target branch to master.

Sure thing I'll do that

**Pull request closed without merging**